src/tooltip/test/tooltip.spec.js

```diff
@@ -67,55 +67,73 @@ describe('tooltip', function() {
       element: '<a data-animation="am-flip-x" bs-tooltip="tooltip">hover me</a>'
     },
     'options-placement-top': {
-      element: '<a data-placement="top" bs-tooltip="tooltip">hover me</a>'
+      element: '<a data-placement="top" bs-tooltip="tooltip" data-viewport="null">hover me</a>'
```

Comment:
Is this really required? Does it means this is a breaking change for any existing implementation?